integration_tests/CMakeLists.txt

```
@@ -242,18 +242,18 @@ RUN(NAME if_05 LABELS gfortran llvm cpp wasm c)
 RUN(NAME while_01 LABELS gfortran llvm cpp x86 wasm c)
 RUN(NAME while_02 LABELS gfortran llvm cpp wasm c)
 
-RUN(NAME doloop_01 LABELS gfortran llvm c cpp x86 wasm)
-RUN(NAME doloop_02 LABELS gfortran llvm c cpp x86 wasm)
+RUN(NAME doloop_01 LABELS gfortran llvm c cpp x86 wasm fortran)
+RUN(NAME doloop_02 LABELS gfortran llvm c cpp x86 wasm fortran)
 RUN(NAME doloop_03 LABELS gfortran llvm c cpp wasm)
-RUN(NAME doloop_04 LABELS gfortran llvm) # uses goto target
+RUN(NAME doloop_04 LABELS gfortran llvm fortran) # uses goto target
 RUN(NAME doloop_05 LABELS gfortran llvm) # uses goto
-RUN(NAME doloop_06 LABELS gfortran llvm c cpp wasm)
-RUN(NAME doloop_07 LABELS gfortran llvm c cpp wasm)
-RUN(NAME doloop_08 LABELS gfortran llvm c cpp wasm NOFAST)
+RUN(NAME doloop_06 LABELS gfortran llvm c cpp wasm fortran)
+RUN(NAME doloop_07 LABELS gfortran llvm c cpp wasm fortan)
```

Here, I think it should be 
```suggestion
RUN(NAME doloop_07 LABELS gfortran llvm c cpp wasm fortran)
```

We should probably throw an error in the Integration tests CMake when non-existent flags are used/provided. This is a separate issue, unrelated to this PR.

---

Yep, sure.

will create an issue for this.